content/guest-essays/utilitarianism-and-climate-change.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Utilitarianism and Climate Change"
+book: "Utilitarianism and Climate Change"
+year: "2023"
+date: 2023-01-01T10:10:55-04:00
+author: "John Broome"
+draft: false
+layout: "book"
+menu: "guest"
+weight: 9
+image: "/img/essays/_______________________.jpg"
```

Comment:
@rychappell we'll want to find a fitting image, perhaps this one?

https://unsplash.com/photos/c7RWVGL8lPA